app/src/main/java/com/bignerdranch/android/playlistmaker/playlist_menu/ui/bottom_sheet/BottomSheetSetuper.kt

```diff
@@ -11,17 +15,21 @@ import com.google.android.material.bottomsheet.BottomSheetDialog
 
 class BottomSheetSetuper(private val activity: Activity?) {
 
+    @SuppressLint("ResourceAsColor")
     fun setupRatio(bottomSheetDialog: BottomSheetDialog, percentage: Float) {
 
         val bottomSheet =
             bottomSheetDialog.findViewById<View>(R.id.design_bottom_sheet) as FrameLayout
+        bottomSheet.setBackgroundTintMode(PorterDuff.Mode.CLEAR);
+        bottomSheet.setBackgroundTintList(ColorStateList.valueOf(Color.TRANSPARENT));
+        bottomSheet.setBackgroundColor(Color.TRANSPARENT);
```

Comment:
🍏 Есть ощущение, что здесь применяется скопированный из Java код, так как остались ";" )
```suggestion
        bottomSheet.backgroundTintMode = PorterDuff.Mode.CLEAR
        bottomSheet.backgroundTintList = ColorStateList.valueOf(Color.TRANSPARENT)
        bottomSheet.setBackgroundColor(Color.TRANSPARENT)
```